Functions/xASL_io_ReadDataPar.m

```diff
@@ -192,3 +203,48 @@
 end
 
 end
+
+%%xASL_io_ReadDataPar_ConvertCellArrays finds all the cell arrays entered as strings and fixes them
```

Comment:
If we normally write json files using spm_jsonwrite, then this shouldn't happen anyway and it's really not according to official JSON format.
The DataPar is one of the only ones that people write by hand and even here I'd try to teach everyone in a tutorial or doc rather the correct way instead of allowing wrong formats.

Reply:
Exactly. Standard is a standard and no need to fix for deviations from that. We only do that for DataPar because we are responsible for providing the wrong instructions previously.

Reply:
OK for writing. When reading JSONs with wrong format, it would be nice if this is possible also for sidecars, not only DataPar, but of course giving a warning. We want users to use a formal JSON format, but remember that if they have a wrong sidecar .JSON, and our script crashes, it can take a while to find this out.

That's why I would make such warnings (and automatic fixes) general.

Unless this is a very specific fix to instructions we provided wrongly in the past.